src/utils/download.js

```diff
@@ -1,5 +1,5 @@
 import * as Virtru from 'virtru-sdk';
-import logAction from './virtruActionLogger.js';
+import logAction from './virtruActionLogger';
```

Comment:
I recommend keeping `.js`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#default_import

Reply:
I agree, but I'd rather be consistent for now. This will become an issue when we start adding `"type":"module"` to our package.json files